editions/tw5.com/tiddlers/HTML Tags.tid

```diff
@@ -0,0 +1,11 @@
+created: 20250211092107689
+modified: 20250211092307574
+tags: Definitions
+title: HTML Tags
+type: text/vnd.tiddlywiki
+
+<<<
+HTML elements are created using ''HTML tags''.
+
+They are grouped by function to help you find what you have in mind easily. An alphabetical list of all elements is provided in the sidebar on every element's page as well as this one.
```

Comment:
I had to read this a few times to understand that the text refers to the link below. We should try to make that clearer.